src/main/java/eu/xenit/gradle/alfrescosdk/tasks/AmpSourceDirectories.java

```diff
@@ -1,10 +1,11 @@
 package eu.xenit.gradle.alfrescosdk.tasks;
 
+import org.gradle.api.file.SourceDirectorySet;
+
 import java.io.File;
 import java.util.Set;
-import org.gradle.api.file.FileTree;
 
-public interface AmpSourceDirectories extends FileTree {
+public interface AmpSourceDirectories extends SourceDirectorySet {
```

Comment:
Since the methods in this interface match what's in `SourceDirectorySet`, this interface can be dropped

Reply:
So we can remove the whole file, you mean?

Reply:
Yes